systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -430,21 +430,25 @@ void verifyLabelsForCRDs() {
 
     void verifyLabelsForKafkaAndZKServices(String clusterName, String appName) {
         LOGGER.info("Verifying labels for Services");
-        List<String> servicesList = new ArrayList<>();
-        servicesList.add(clusterName + "-kafka-bootstrap");
-        servicesList.add(clusterName + "-kafka-brokers");
-        servicesList.add(clusterName + "-zookeeper-nodes");
-        servicesList.add(clusterName + "-zookeeper-client");
+        String kafkaServiceName = clusterName + "-kafka";
+        String zookeeperServiceName = clusterName + "-zookeeper";
 
-        for (String serviceName : servicesList) {
+        Map<String, String> servicesList = new HashMap<>();
```

Comment:
It's not servicesList but map )
```suggestion
        Map<String, String> servicesMap = new HashMap<>();
```